app/controllers/issues_controller.rb

```diff
@@ -22,7 +23,7 @@ def new
   end
 
   def show
-    @issue = Issue.find(params[:id])
+    @voted = @issue.votes.where(user_id: current_user.id)
     load_relevant_issues
     redirect_back(fallback_location: root_path) unless \
                                                 @issue.can_read?(current_user)
```

Comment:
@Kmalynss I think yes, it is possible
But then should I apply any changes to upvote and downvote methods?